api/v1alpha1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2021.
```

Comment:
Not sure? It was generated by the SDK, we don't have anything at the moment https://github.com/nginxinc/nginx-ingress-operator/blob/master/pkg/apis/k8s/v1alpha1/register.go

Reply:
I'm not sure what would be the problem with only some files with the Copyright, since it was autogenerated I guess we can add it as tech debt if we want to remove it/change it later.

Reply:
it is not a big issue, just consistency.  But I don't see why not just remove it - since in this PR we're adding auto-generated stuff, so why not just edit that stuff? Why do we need to create a tech debt tasks? anyway, this is not a blocker